node/tests/node.rs

```diff
@@ -28,7 +28,8 @@ async fn connects_to_the_go_bridge_node() {
     let node = new_connected_node().await;
 
     let info = node.network_info().await.unwrap();
-    assert_eq!(info.num_peers(), 1);
+    // our docker setup is two bridges connected, so there's already one another node connected
+    assert_eq!(info.num_peers(), 2);
```

Comment:
may be more future proof
```suggestion
    assert!(info.num_peers() >= 1);
```